crates/pallet-bridges-initializer-currency-swap/src/mock.rs

```diff
@@ -0,0 +1,206 @@
+    type AccountStore = System;
+    type MaxLocks = ();
+    type MaxReserves = ();
+    type ReserveIdentifier = [u8; 8];
+    type WeightInfo = ();
+}
+
+impl pallet_balances::Config<BalancesInstanceEvm> for Test {
+    type Balance = u64;
+    type RuntimeEvent = RuntimeEvent;
+    type DustRemoval = ();
+    type ExistentialDeposit = ConstU64<EXISTENTIAL_DEPOSIT_EVM>;
+    type AccountStore = StorageMapShim<
+        pallet_balances::Account<Test, BalancesInstanceEvm>,
+        frame_system::Provider<Test>,
+        AccountId,
+        pallet_balances::AccountData<Balance>,
+    >;
+    type MaxLocks = ();
+    type MaxReserves = ();
+    type ReserveIdentifier = [u8; 8];
+    type WeightInfo = ();
+}
+
+parameter_types! {
+    pub const SwapBridgeNativeToEvmPotPalletId: PalletId = PalletId(*b"humanoNE");
+    pub const SwapBridgeEvmToNativePotPalletId: PalletId = PalletId(*b"humanoEN");
+}
+
+type PotInstanceSwapBridgeNativeToEvm = pallet_pot::Instance1;
+type PotInstanceSwapBridgeEvmToNative = pallet_pot::Instance2;
+
+impl pallet_pot::Config<PotInstanceSwapBridgeNativeToEvm> for Test {
+    type RuntimeEvent = RuntimeEvent;
+    type AccountId = AccountId;
+    type PalletId = SwapBridgeNativeToEvmPotPalletId;
+    type Currency = Balances;
+}
+
+impl pallet_pot::Config<PotInstanceSwapBridgeEvmToNative> for Test {
+    type RuntimeEvent = RuntimeEvent;
+    type AccountId = EvmAccountId;
+    type PalletId = SwapBridgeEvmToNativePotPalletId;
+    type Currency = EvmBalances;
+}
+
+parameter_types! {
+    pub const SwapBridgeNativeToEvmPalletId: PalletId = PalletId(*b"hmsb/ne1");
+    pub const SwapBridgeEvmToNativePalletId: PalletId = PalletId(*b"hmsb/en1");
+}
+
+parameter_types! {
+    pub SwapBridgeNativeToEvmPotAccountId: AccountId = SwapBridgeNativeToEvmPot::account_id();
+    pub SwapBridgeEvmToNativePotAccountId: AccountId = SwapBridgeEvmToNativePot::account_id();
+}
+
+parameter_types! {
+    pub NativeTreasury: AccountId = 4200;
+}
+
+pub struct NativeBridgeBalanceMaker;
+
+impl BalanceMaker<AccountId, Balances> for NativeBridgeBalanceMaker {
+    const IS_SWAPPABLE_CHANGED: bool = true;
+
+    fn make_balance(amount: Balance) -> Result<(), DispatchError> {
+        let imbalance = Balances::withdraw(
+            &NativeTreasury::get(),
+            amount,
+            frame_support::traits::WithdrawReasons::TRANSFER,
+            frame_support::traits::ExistenceRequirement::KeepAlive,
+        )?;
+
+        Balances::resolve_creating(&SwapBridgeNativeToEvmPotAccountId::get(), imbalance);
+        Ok(())
+    }
+}
+
+pub struct EvmBridgeBalanceMaker;
+
+impl BalanceMaker<AccountId, EvmBalances> for EvmBridgeBalanceMaker {
+    const IS_SWAPPABLE_CHANGED: bool = false;
+
+    fn make_balance(amount: Balance) -> Result<(), DispatchError> {
+        let imbalance = EvmBalances::issue(amount);
+
+        EvmBalances::resolve_creating(&SwapBridgeNativeToEvmPotAccountId::get(), imbalance);
+        Ok(())
+    }
+}
```

Comment:
Let's drop these and just inline the necessary operations. It would just be simpler, and less knobs to test

Reply:
Dropped - https://github.com/humanode-network/humanode/pull/716/commits/8c0c96be4f200560e1ea9fb782a324213b54689c